db/seeds.rb

```diff
@@ -124,13 +124,13 @@
                         brand: 'K2',
                         image: File.open('app/assets/images/fastplant.png')
                       },
-                      { name: 'Burton Process Snowboard',
+                      { name: 'Burton Process',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.